lib/App/index.js

```diff
@@ -285,10 +285,31 @@ class App {
           await this._validateImages(driver.images, 'driver');
         }
 
+        // validate `appJson.drivers[].platforms`
+        if (driver.platforms && appJson.platforms) {
```

Comment:
Do we need to check if `driver.platforms` is an array before using `includes()`? e.g. throw "driver.platforms must be an array"

Reply:
(same for connectivity perhaps)